go/host/db/blocks.go

```diff
@@ -0,0 +1,107 @@
+package db
+
+import (
+	"bytes"
+
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethdb"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/obscuronet/go-obscuro/go/common/log"
+)
+
+// DB methods relating to batches.
+
+// headerKey = blockHeaderPrefix  + hash
+func blockHeaderKey(hash gethcommon.Hash) []byte {
```

Comment:
typically private methods are after the public ones

Reply:
Fixing.